js/miniautures.js

```diff
@@ -0,0 +1,31 @@
+const pictures = document.querySelector ('.pictures');
+const templateFragment = document.querySelector ('#picture').content;
+const template = templateFragment.querySelector('a');
+const fragment = document.createDocumentFragment();
+
+for (let i = 0; i < /*???*/; i++) {
+    const newPhoto = template.cloneNode(true); // Клонируем элемент со всеми "внутренностями"
+    newPhoto.img.src = import { getPhotoUrl } from "./data";
+    newPhoto.pictures__likes = import { getNewPhoto.likes } from "./data";
+    newPhoto.pictures__comments = import MAX_COMMENTS_COUNT;
+    fragment.appendChild(newPhoto);
+}
+
+pictures.append(fragment);
+
```

Comment:
Код отрисовки примерно верный, но идея не совсем верна: нужно один раз сгененрировать данные, а потом их уже использовать и отрисовывать именно их, предлагаю, например, такой вариант, вместо твоего

```suggestion
const drawPicture = funcition (description) {
    const newPhoto = template.cloneNode(true); // Клонируем элемент со всеми "внутренностями"
    
    // Заполняем нашими данными
    newPhoto.img.src = description.url ; 
    newPhoto.pictures__likes = description.likes;
    newPhoto.pictures__comments = description.comments.length;
    
    return newPhoto;
}

export const drawPictures = funcition (descriptions) {
const fragment = document.createDocumentFragment();
  descriptions.forEach(description => {
    const pictureEl = drawPicture(description) // для каждого описания создаешь изображение
       fragment.appendChild(pictureEl );
  })
 pictures.append(fragment);
} 
```